v-next/hardhat/src/internal/builtin-plugins/solidity-test/formatters.ts

```diff
@@ -133,7 +135,14 @@ function formatTrace(trace: CallTrace): NestedArray<string> {
     }
   }
   if (formattedOutputs !== undefined) {
-    closingLine = `${color("←")} ${formattedOutputs}`;
+    if (
+      formattedOutputs === "EvmError: Revert" ||
+      formattedOutputs.startsWith("revert:")
+    ) {
+      closingLine = `${color("←")} ${color("[Revert]")} ${formattedOutputs}`;
+    } else {
+      closingLine = `${color("←")} ${formattedOutputs}`;
+    }
```

Comment:
> Reverts are not indicated as clearly as in Foundry. Notice the missing [Revert]:

I addressed this suggestion here. So far, I noticed these two patterns that suggest we're dealing with a revert. We can expand and/or formalise this more as needed in the future.